lib/Indexer/Adapter/Tolerant/Indexer/ClassDeclarationIndexer.php

```diff
@@ -35,9 +37,28 @@ public function index(Index $index, TextDocument $document, Node $node): void
         $this->indexClassInterfaces($index, $record, $node);
         $this->indexBaseClass($index, $record, $node);
 
+        $this->indexAttributes($record, $node);
+
         $index->write($record);
     }
 
+    public function indexAttributes(ClassRecord $record, ClassDeclaration $node): void
+    {
+        $attributes = $node->attributes ?? [];
+        if (count($attributes) === 0) {
+            return;
+        }
+
+        foreach ($attributes as $attributeGroup) {
+            foreach ($attributeGroup->attributes->children as $attribute) {
+                /** @var Attribute $attribute */
+                if ($attribute->name->getText() === 'Attribute') {
+                    $record->addFlag(HasFlags::ATTRIBUTE);
```

Comment:
can continue here, no need to go through all the attributes.

Reply:
True, one other thing I also wanted to bring up is, this only checks the name of the attribtue not the class. Which means that it returns true on any class called Attribute no matter the namespace.

Reply:
`$attribute->getResolvedName()` should return the FQN;